libcextract/FunctionDepsFinder.hh

```diff
@@ -43,6 +43,33 @@ class ClosureSet
     Dependencies.insert(decl);
   }
 
+  /*
+   * Check if the same Decl was already added into the closure but for a
+   * different typename.
+   */
+  TypedefDecl *insideRangeOfDecl(TypedefDecl *decl)
+  {
+    SourceRange range = decl->getSourceRange();
+
+    for (auto it = Dependencies.begin(); it != Dependencies.end(); ++it) {
+      if (TypedefDecl *cur_decl = dynamic_cast<TypedefDecl *>(*it)) {
```

Comment:
Please avoid using dynamic_cast<type*> and use dyn_cast. This works on clang builds with RTTI disabled.

Reply:
I just copied this line from the FunctionDepsFinder :P

Thanks for the suggestion